service-registry/src/test/java/org/apache/servicecomb/serviceregistry/task/TestMicroserviceRegisterTask.java

```diff
@@ -498,12 +508,36 @@ public void testLocalSchemaAndServiceCenterSchemaDiff(@Mocked ServiceRegistryCli
         result = otherMicroservice;
         srClient.getSchemas(anyString);
         result = onlineSchemasHolder;
+        srClient.getSchema(anyString, anyString);
+        result = "abcdefghijklmn";
       }
     };
 
-    microservice.addSchema("s1", "abdc");
+    microservice.addSchema("s1", "abcdefghijklmnopqrstuvwxyz");
     microservice.setEnvironment("prod");
     MicroserviceRegisterTask registerTask = new MicroserviceRegisterTask(eventBus, srClient, microservice);
-    registerTask.run();
+
+    Logger logger = Logger.getLogger(MicroserviceRegisterTask.class);
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
+    Layout layout = new SimpleLayout();
+    Appender appender = new WriterAppender(layout, out);
+    logger.addAppender(appender);
+    Boolean isIlleagalException = false;
+
+    try {
+      registerTask.run();
+    } catch (IllegalStateException exception) {
+      isIlleagalException = true;
+      String logMsg = out.toString();
+      assertNotNull(logMsg);
+      Assert.assertTrue(logMsg.contains("service center schema:"));
```

Comment:
it's better not use contains,  but use equals

Reply:
OK